webots_ros2_core/webots_ros2_core/devices/laser_device.py

```diff
@@ -38,15 +38,20 @@ class LaserDevice(SensorDevice):
         wb_device (Lidar): Webots node of type Lidar.
 
     Kwargs:
-        params (dict): Inherited from `SensorDevice`
+        params (dict): Inherited from `SensorDevice` + the following::
```

Comment:
```suggestion
        params (dict): Inherited from `SensorDevice` + the following:
```

Reply:
According to Sphinx, `::` is used for "literal blocks":
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
I thought this was a typo :wink: